tools/deploy

```diff
@@ -2,6 +2,10 @@
 
 # Track Version
 git log -n 1 > VERSION
+# Ensure we have the correct node & npm versions
```

Comment:
Just a thought -- this could mean the pre-npm-install/setup we have in Docker is now ignored. Just an optimization opportunity for down the road.